### PR TITLE
:robot: Add bytecodeviewer.vm

### LINKED ISSUE #76: Package proposal: bytecodeviewer.vm
### Package Name

bytecodeviewer

### Tool Name

Bytecode Viewer

### Tool's version number

2.11.2

### Category

Java

### Tool's authors

Konloch

### Tool's description

A lightweight user-friendly Java/Android Bytecode Viewer, Decompiler and more.

### Package type

ZIP_EXE

### Download URL

https://github.com/Konloch/bytecode-viewer/releases/download/v2.11.2/Bytecode-Viewer-2.11.2.jar

### Download SHA256 Hash

536ad387424106083f76cd0cb7c051a22aff21f08663ba2539c11f1ddef9147f

### Why is this tool a good addition?

To analyze Java/Android files.

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/76 :rocket:

Closes #76

### CODE REVIEW

**Comment by Ana06** [comment]
Unsupported package type, see https://github.com/mandiant/VM-Packages/issues/76#issuecomment-1250760598

**Comment by mr-tz** [comment]
😞

**Comment by MalwareMechanic** [comment]
Closes #76

**Review by mr-tz — APPROVED** [review]

File: packages/bytecodeviewer.vm/bytecodeviewer.vm.nuspec [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <description>A lightweight user-friendly Java/Android Bytecode Viewer, Decompiler and more.</description>
     <dependencies>
       <dependency id="common.vm" />
+      <dependency id="javaruntime" />
```
Comment (mr-tz) [inline packages/bytecodeviewer.vm/bytecodeviewer.vm.nuspec:10]: should this be a fixed version?

**Pull request merged**